lapce-ui/src/svg.rs

```diff
@@ -85,8 +106,13 @@ pub fn file_svg(path: &Path) -> Svg {
             })
             .unwrap_or("default_file.svg")
     };
-
-    get_svg(file_type).unwrap()
+    
+    let svg_file = get_svg(file_type);
+    if svg_file.is_none() {
+        panic!("Missing svg icon '{}'", file_type);
```

Comment:
Same as above: prefer `.expect()` (or in this case, `unwrap_or_else(|| panic!())`) over an `if`